pages/index.mdx

```diff
@@ -10,12 +10,13 @@ import { GridRow, GridColumn } from '@components/GridLayout'
 
 ## Deploy applications without infrastructure specialists
 
-You can get something up on the Internet in minutes, without depending on a webops team.
+You can get something up on the internet in minutes, without depending on a web operations team.
```

Comment:
```suggestion
You can deploy your app to the internet in minutes, without depending on a web operations team.
```

Reply:
Webops / Web Operations team is a term that should go away

Reply:
without depending on a team of site reliability engineers

?

Reply:
yep, sounds good